workitem/field_definition.go

```diff
@@ -235,5 +235,5 @@ func compatibleFields(existing FieldDefinition, new FieldDefinition) bool {
 	if existing.Required != new.Required {
 		return false
 	}
-	return reflect.DeepEqual(existing.Type, new.Type)
+	return existing.Type.GetKind() == new.Type.GetKind()
```

Comment:
@aslakknutsen I needed to simplify this comparison to only check the Kind of the type or otherwise I wouldn't be able to overwrite the values of the `system.state` field in a WIT from the Scrum template. Do you think it is fine this way or is the comparison too relaxed? I have to check other tests as well but without this comparison I cannot overwrite.

Reply:
Because a DeepEqual would compare the configured Enum values as well?

Reply:
Yes, that is the reason.